yarn.lock

```diff
@@ -752,6 +752,11 @@
   resolved "https://registry.yarnpkg.com/@fiatconnect/fiatconnect-types/-/fiatconnect-types-6.0.0.tgz#ce458140a5af589248460c5270e29aeddc27cf28"
   integrity sha512-rBYe2Mbjyw6WlP6GY2BzY7jac/aP4VxccDJzbCFkx17aYqj724Bj2yLhBeF9znL44sQoAuYLJeyT0f80xGQR2w==
 
+"@fiatconnect/fiatconnect-types@^6.0.1":
```

Comment:
can you do a yarn-deduplicate? seems like we have both 6.0.0 and 6.0.1 here

Reply:
fiatconnect-sdk also imports fiatconnect-types but still has it at v6.0.0 so both are included currently. I'll update the sdk first, then update this

Reply:
Nevermind, not necessary